.github/workflows/apiserver-test.yml

```diff
@@ -180,7 +180,9 @@ jobs:
           make e2e-apiserver-test
 
       - name: Stop kubevela, get profile
-        run: make end-e2e-core
+        run: |
+          make end-e2e-core
+          CORE_NAME=kubevela-shard sh ./hack/e2e/end_e2e_core.sh
```

Comment:
Since `end-e2e-core` is a makefile target, It'd be better if we wrap `CORE_NAME=kubevela-shard sh ./hack/e2e/end_e2e_core.sh` into another one to keep style here consistent.

Reply:
add `make end-e2e-core-shards`